app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/dagger/component/AppComponent.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.marcinwisniewski.intivetrainingapp.dagger.component
+
+import android.app.Application
+import com.example.marcinwisniewski.intivetrainingapp.dagger.module.AppModule
+import com.example.marcinwisniewski.intivetrainingapp.dagger.module.FragmentModule
+import com.example.marcinwisniewski.intivetrainingapp.dagger.module.NetworkModule
+import com.example.marcinwisniewski.intivetrainingapp.main.MovieApplication
+import dagger.BindsInstance
+import dagger.Component
+import dagger.android.AndroidInjectionModule
+import dagger.android.support.AndroidSupportInjectionModule
+import javax.inject.Singleton
+
```

Comment:
Here is a standard component creation - I am adding my modules and android injection module -thanks to that I am able to do an injection also in my fragments